tests/driver_at86rf2xx_aes/Makefile

```diff
@@ -1,6 +1,7 @@
 include ../Makefile.tests_common
 
 DISABLE_MODULE += auto_init_at86rf2xx
+BOARD_BLACKLIST := native
```

Comment:
I think we should rather use `TEST_ON_CI_WHITELIST += samr21-xpro`.

Reply:
but is that enough for `compile_and_test_for_board.py`?

Reply:
Ah yes. With #15771, yes :)
Simply blacklist the board like here would also skip the build, but that is not the problem.

Reply:
ok, I will do that then

Reply:
hmmm I just noticed `TEST_ON_CI_WHITELIST` is already used here :/